src/uu/lsmem/src/lsmem.rs

```diff
@@ -475,9 +503,46 @@ fn print_table(lsmem: &Lsmem, opts: &Options) {
     table.with(Disable::column(ByColumnName::new("NODE")));
     table.with(Disable::column(ByColumnName::new("ZONES")));
 
+    if opts.noheadings {
+        table.with(Disable::row(Rows::first()));
+    }
+
     println!("{table}");
 }
 
+fn print_json(lsmem: &Lsmem, opts: &Options) {
+    let table_json = TableRowJson {
+        memory: create_table_rows(lsmem, opts),
+    };
+
+    let table_json_string = serde_json::to_string_pretty(&table_json).unwrap();
+    println!("{table_json_string}");
+}
+
+fn print_pairs(lsmem: &Lsmem, opts: &Options) {
+    let table_rows = create_table_rows(lsmem, opts);
+    let mut table_pairs_string = String::new();
+    for row in table_rows {
+        table_pairs_string += &row.to_pairs_string();
+        table_pairs_string += "\n";
+    }
+    // remove the last newline
+    table_pairs_string.pop();
+    println!("{table_pairs_string}");
+}
+
+fn print_raw(lsmem: &Lsmem, opts: &Options) {
+    let table_rows = create_table_rows(lsmem, opts);
+    let mut table_raw_string = String::new();
+    for row in table_rows {
+        table_raw_string += &row.to_raw_string();
+        table_raw_string += "\n";
+    }
+    // remove the last newline
+    table_raw_string.pop();
+    println!("{table_raw_string}");
```

Comment:
The header for the raw string is missing. The original `lsmem` outputs the following header:
```
$ lsmem -r
RANGE SIZE STATE REMOVABLE BLOCK
```